opencensus/proto/metrics/metrics.proto

```diff
@@ -147,13 +147,10 @@ message Point {
   }
 }
 
-// Distribution contains summary statistics for a population of values. It
-// optionally contains a histogram representing the distribution of those
-// values across a set of buckets.
-message DistributionValue {
+// Histogram contains summary statistics for a population of values.
```

Comment:
In this case histogram buckets are required for `HistogramValue`?
What if users only want the summary stats (sum, count, mean, etc.) without a histogram?

Reply:
SGTM